src/infra/core/src/testing/minio_server.rs

```diff
@@ -113,7 +113,7 @@ impl LocalS3Server {
         let minio = MinioServer::new(tmp_dir.path(), access_key, secret_key).await;
 
         let url = Url::parse(&format!(
-            "s3+http://{}:{}/{}",
+            "s3+http://{}:{}/{}/",
```

Comment:
This is needed only to make tests more succinct. As sync service normalizes URLs to always have the trailing slash adding it here allows direct comparisons between these URLs and those is all kinds of errors.